index.bs

```diff
@@ -11391,15 +11392,56 @@ The algorithm for rendering a block of audio from a {{BaseAudioContext}}
 <em>Q</em> is comprised of multiple steps and explained in further detail
 in the algorithm of <a href="#rendering-a-graph">rendering a graph</a>.
 
-<div class="note">
-	In practice, the {{AudioContext}} <a>rendering thread</a> is
-	often running off a system-level audio callback, that executes in
-	an isochronous fashion.
-
-	An {{OfflineAudioContext}} is not required to have a
-	system-level audio callback, but behaves as if it did with the
-	callback happening as soon as the previous callback is
-	finished.
+<div class="correction proposed" id="c2475-1">
+	<span class="marker">Proposed Addition
+		<a href="https://github.com/WebAudio/web-audio-api/issues/2475">Issue
+			2375</a>.
+	</span>
+	Add definition for various concepts related to system-level audio callbacks
+	<div class="amendment-buttons">
+		Buttons here
+	</div>
+	<ins cite=#c2475>
+		The {{AudioContext}} <a>rendering thread</a> is driven by a
+		<dfn>system-level audio callback</dfn>, that is periodically invoked in an
+		isochronous fashion. This <a>system-level audio callback</a> is called
+		repeatedly, and each call has a <dfn>system-level audio callback buffer
+		size</dfn> (that is potentially variable between calls), which is the
+		number of sample-frames that needs to be computed on time before the
+		<a>system-level audio callback</a> returns.
+
+		A <dfn>load value</dfn> is computed for each <a>system-level audio
+			callback</a>, by dividing its execution duration by the <a>system-level
+			audio callback buffer size</a> divided by {{BaseAudioContext/sampleRate}}.
+
+		The <a>load value</a> usually is below 1.0, meaning that it took less time
```

Comment:
Perhaps:

Ideally the load value is below 1.0, meaning that the renderer was able to finish the callback before the given callback interval. When the load value goes beyond 1.0, it means the system could not render audio fast enough resulting in a "buffer underrun".

Reply:
This is problematic, because intervals aren't necessarily regular (e.g. on Linux/PulseAudio, or Windows sometimes). This really is a matter of rendering it in less time that it takes to output.